### DISCUSSION TITLE
Only 3 models used in `.compare_models` when using `TSForecastingExperiment`

### DISCUSSION THREAD

**Discussion by jmakov** [pycaret/pycaret#3770] [Q&A] [answered]
When I run `compare_models()` for time series the output only shows 3 models: croston, naive, grand_means. From the docs I expected a lot more models being evaluated. Not sure what I'm missing.

pycaret version 3.1.0, installed with `pycaret[full]`.

**Comment by ngupta23** [comment, marked as answer]
It is possible that the models are failing to train (for various reasons). By default these are ignored so that the code execution does not break. You can change argument `errors = 'ignore'` to  `errors = 'raise'` to see where the error is originating from.

https://pycaret.readthedocs.io/en/latest/api/time_series.html#pycaret.time_series.compare_models

**Comment by jmakov** [comment]
The `errors="raise"` helped me to find the problem - the "timestamp" column was included in the training data. Dropping it solves the problem, thank you very much!
